src/lib/apiClient.ts

```diff
@@ -0,0 +1,37 @@
+class ApiClient {
+  async fetchDocs(endpoint: string) {
+    const response = await fetch('/api/graphql_proxy/docs', {
+      method: 'POST',
+      body: JSON.stringify({ endpoint }),
+    });
+
+    return response.json();
+  }
+
+  async runQuery({
+    query,
+    endpoint,
+    headers,
+    variables,
+  }: {
+    query: string;
+    endpoint: string;
+    headers: object;
+    variables: object;
+  }) {
+    const response = await fetch('/api/graphql_proxy/query', {
+      method: 'POST',
+      body: JSON.stringify({
+        query,
+        endpoint,
+        headers,
+        variables,
+      }),
+    });
+    return response.json();
+  }
+}
```

Comment:
I would suggest to pick everything from class out, beause there is no need to use classes at all. You do not provide to constructor and do not store any data within class. Nothing will change tho. Only method calls without class instance :)